pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -147,28 +150,54 @@ func NewSchedulingController(
 		}, placementDecisionInformer.Informer()).
 		WithBareInformers(clusterInformer.Informer(), clusterSetInformer.Informer(), clusterSetBindingInformer.Informer()).
 		WithSync(c.sync).
+		ResyncEvery(ResyncInterval).
```

Comment:
I suspect that a full requeue every five minutes will become too heavy.  I suggest adding a comment that if latency for placements becomes an issue and if greater parallelization doesn't address the problem, that two different controllers could be written, one for reactive changes and one for resyncing the entire list.  This would allow responsiveness.

We did this for resourcequota in kube.

Reply:
Thanks for the comment, remove resync() from this PR, will create another PR to handle that.